sig-multicluster/management-cluster-position-statement.md

```diff
@@ -0,0 +1,46 @@
+# Management Cluster - SIG Multicluster Position Statement
```

Comment:
I feel like the term`Management` is a bit too generic. Could we do a poll first to gather some community feedback?

Reply:
This is a great idea, thank you! I'll ask in Slack for a list and try to figure out where/how to run said poll :)